roles/foreman_client_repositories/tasks/main.yml

```diff
@@ -0,0 +1,16 @@
+---
+- name: 'Foreman Client Staging Repository'
+  yum_repository:
+    name: foreman-client-koji
+    description: "Foreman {{ foreman_client_repositories_version }} Client Koji Repository"
+    baseurl: "http://koji.katello.org/releases/yum/foreman-client-{{ foreman_client_repositories_version }}/el{{ ansible_distribution_major_version }}/x86_64/"
+    priority: 1
+    gpgcheck: 0
+  when: foreman_client_repositories_environment == "staging"
+
+- name: 'Setup Foreman Client Release Repository'
+  yum:
+    name: https://fedorapeople.org/groups/foreman/releases/yum/{{ foreman_client_repositories_version }}/client/el{{ ansible_distribution_major_version }}/x86_64/foreman-client-repos-latest.rpm
```

Comment:
Is it allowed to have multiple keys with the same name? Doesn't look intended though.